css/styles.css

```diff
@@ -14,10 +14,28 @@
  body {
     background-color: #fff;
     color: #333;
-    font-family: Helvetica Neue, Helvetica, Arial;
-    font-weight: normal;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 300;
  }
 
+ h1, h2, h3, h4, h5, h6 {
+    font-weight: 400;
+ }
+
+form input[type="text"] {
+    margin: 20px 20px 20px 0;
+    padding: 5px 40px 5px 20px;
```

Comment:
This padding is more likely to be `5px` for any side and to make it longer apply some width.